src/main/kotlin/com/atlassian/performance/tools/report/api/judge/RelativeNonparametricPerformanceJudge.kt

```diff
@@ -61,7 +61,7 @@ class RelativeNonparametricPerformanceJudge(
             )
         val test = ShiftedDistributionRegressionTest(baseline, experiment, mwAlpha = significance, ksAlpha = 0.0)
         return if (test.isExperimentRegressed(toleranceRatio.toDouble())) {
-            val confidenceLevelPercent = (1.0 - significance).toPercentage(decimalPlaces = 2)
+            val confidenceLevelPercent = (1.0 - significance).toPercentage(decimalPlaces = 0, includeSign = false)
```

Comment:
Nice catch, it's gonna be less confusing now. It's the little things that hurt the most :)